src/OmniSharp.DotNetTest/Services/RunTestsInClassService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+using OmniSharp.DotNetTest.Models;
+using OmniSharp.Eventing;
+using OmniSharp.Mef;
+using OmniSharp.Services;
+
+namespace OmniSharp.DotNetTest.Services
+{
+    [OmniSharpHandler(OmniSharpEndpoints.V2.RunAllTestsInClass, LanguageNames.CSharp)]
+    internal class RunTestsInClassService : BaseTestService<RunTestsInClassRequest, RunTestResponse[]>
```

Comment:
How similar is this to the class that runs a single test in a class?

Reply:
It is the same now, only the type of request differs. For one it is RunTestRequest and for the other it is RunTestInClassRequest and the corresponding fields - `MethodName` and `MethodNames` also differ.